src/widgets/Grabber.h

```diff
@@ -46,15 +46,13 @@ class GrabberEvent:public wxCommandEvent
    GrabberEvent(wxEventType type = wxEVT_NULL,
                 wxWindowID winid = 0,
                 const wxPoint& pt = wxDefaultPosition)
-   : wxCommandEvent(type, winid)
+   : wxCommandEvent(type, winid), mPos(pt)
```

Comment:
That documentation is pretty old, might be that compilers are a lot smarter than they were back when that document was written… I guess it is not an optimization then, only a style change.